src/Common/SelectInterface.php

```diff
@@ -79,6 +88,35 @@ public function cols(array $cols);
 
     /**
      *
+     * Remove a column via its alias.
+     *
+     * @param string $alias The column to remove
+     *
+     * @return bool
+     *
+     */
+    public function removeCol($alias);
+
+    /**
+     *
+     * Does the query have any columns in it?
+     *
+     * @return bool
+     *
+     */
+    public function hasCols();
```

Comment:
yes. i can see why it was added (so you can check if that part of the query is valid). What about adding `hasCol($name_or_alias)`?